webpack-sentry-plugin.js

```diff
@@ -21,6 +21,22 @@ module.exports = {
     options: { pluginOptions, razzleOptions, webpackOptions },
     paths,
   }) {
+    if (
+      process.env.RAZZLE_SENTRY_AUTH_TOKEN &&
```

Comment:
As all `RAZZLE_` environment variables are exposed also to client window.env. we should consider renaming this sensitive env and not expose it to the client.